tests/builds/experiments_ymls/build_examples/experiments.yml

```diff
@@ -44,12 +58,12 @@ builds:
 revisions:
   - name: git-gdsb
     build_version:
-      'test-git-gdsb': '027ee5048e8c00648185ee4321dab969a80396fe' #arbitrary commit hash to ensure determinism
+      'test-git-gdsb': '3fe8b2e38d9de38b459e2e958df66577b77ccccf' #arbitrary commit hash to ensure determinism
```

Comment:
I guess the comment is a bit... confusing. It's not determinism that we want but reproducibility.